modules/core/shared/src/test/scala/gem/laws/SectionLaws.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.laws
+
+import cats.Eq
+import cats.implicits._
+import gem.util.Section
+
+final case class SectionLaws[A, B](fab: Section[A, B]) {
+
+  def normalize(a: A): IsEq[B] =
+    fab.get(fab.normalize(a)) <-> fab.get(a)
```

Comment:
What's `<->`? From cats?

Reply:
Rob wrote it, but I wonder if it is better than using the existing `<==>` from `IsEqOps`?  Or maybe that doesn't apply for some reason?

```
package gem

package object laws {

  type IsEq[A] = cats.kernel.laws.IsEq[A]
  val IsEq: cats.kernel.laws.IsEq.type = cats.kernel.laws.IsEq

  implicit final class IsEqArrow[A](val lhs: A) extends AnyVal {
    def <->(rhs: A): IsEq[A] = IsEq(lhs, rhs)
  }
  
}
```

Reply:
I defined this so our law-checking stuff would have exactly the same structure used by cats, which defines this alias and syntax the same way. There are so many arbitrary choices in testing setups that I thought we should just copy what cats is doing and trust that the decisions taken there were sensible.